pages/linux/nproc.md

```diff
@@ -0,0 +1,15 @@
+# nproc
+
+> Print the number of processing units (normally CPUs) available.
+
+- Display the number of available processing units:
+
+`nproc`
+
+- Display the number of installed processing units, including any inactive ones:
+
+`nproc --all`
+
+- If possible, subtract a given number of units from the returned value:
```

Comment:
Just curious to know why would this flag be helpful ? Can I not subtract the result anyways ?

Reply:
True, but in a script you'd have to ensure that you don't go below 1 with some fiddly nonsense - which this does automagically for you :P